templates/new/stages.py.txt

```diff
@@ -1,19 +1,19 @@
-from surround import Estimator, SurroundData, Validator
+from surround import Estimator, State, Validator
 
 
-class {safe_project_name}Data(SurroundData):
+class AssemblerState(State):
     input_data = None
     output_data = None
 
 
-class ValidateData(Validator):
-    def validate(self, surround_data, config):
-        if not surround_data.input_data:
+class InputValidator(Validator):
```

Comment:
Can we please remove the `stages.py` file? This should be replaced by a `stages` package that contains a separate file for each of the classes in this folder. `AssemblerState` can be its own class in this directory.

Reply:
Done :) also made `stage.py` in Surround a package (I read this comment wrong), but it may be something we want to keep in anyway?